elytron-oidc-client/src/main/resources/org/wildfly/extension/elytron/oidc/LocalDescriptions.properties

```diff
@@ -198,9 +199,11 @@ elytron-oidc-client.secure-server.credential.client-key-alias=The client key ali
 elytron-oidc-client.secure-server.credential.token-timeout=The amount of time after which the token expires and can no longer be used to authenticate requests.
 elytron-oidc-client.secure-server.credential.secret=The client secret that was registered with the OpenID provider.
 elytron-oidc-client.secure-server.credential.algorithm=The credential signature algorithm used by the OpenID provider.
+elytron-oidc-client.secure-server.credential.scope=The scope values are used to request specific or additional permissions for OIDC.
 elytron-oidc-client.secure-server.redirect-rewrite-rule.add=Add the rewrite rule for the redirect URI. The rewrite rule is an object notation, where the key is a regular expression with which the redirect URI is matched and the value is the replacement String. You can use the '$' character to use for back-reference in the replacement String.
 elytron-oidc-client.secure-server.redirect-rewrite-rule.remove=Remove the rewrite rule for the redirect URI.
 elytron-oidc-client.secure-server.redirect-rewrite-rule.replacement=The replacement String in the rewrite rule for the redirect URI.
+elytron-oidc-client.secure-server.scope=The scope values are used to request specific or additional permissions for OIDC.
```

Comment:
We should update the analysis document to mention that the scope attribute is also being added to the `secure-server` resource.

Reply:
Could reword this to: A space separated list of scope values that should be used in the OIDC authentication request. Scopes can be used to request that specific sets of information be made available as Claim Values.